docs/builder_image.md

```diff
@@ -99,7 +99,7 @@ The `run` script is responsible for executing your application.
 ## save-artifacts
 
 The `save-artifacts` script is responsible for gathering all the dependencies which
-existence can speed up the following build processes (eg. for Ruby - Gemfiles,
+existence can speed up the following build processes (eg. for Ruby - Gemfile,
```

Comment:
still not correct, in Ruby, we archive gems installed by Bundler, the Gemfile is only configuration, what we really do is we archive Gemfile.lock.... I would avoid mentioning the Gemfile at all ;-) just say for Ruby we archive gems installed by Bundler ;-)